src/Cornelis/Types/Agda.hs

```diff
@@ -318,21 +283,22 @@ data HighlightingLevel
     -- ^ This includes both non-interactive highlighting and
     -- interactive highlighting of the expression that is currently
     -- being type-checked.
-    deriving (Eq, Ord, Show, Read, Data, Generic)
+    deriving (Eq, Ord, Show, Read, Generic)
 
 data HighlightingMethod
   = Direct
     -- ^ Via stdout.
   | Indirect
     -- ^ Both via files and via stdout.
-    deriving (Eq, Show, Read, Data, Generic)
+    deriving (Eq, Show, Read, Generic)
 
 
 data Remove
   = Remove
   | Keep
   deriving (Show, Read)
 
+{-
```

Comment:
Do we need to keep this?

Reply:
no; but we would like _show_ instances, or something equivalent, that lines up with this logic

Reply:
feel free to remove and i can get those in later